action.yml

```diff
@@ -45,6 +45,13 @@ inputs:
     default: auto
   pre-test-cmd:
     description: Extra command to invoke before ansible-test
+  pull-request-change-detection:
+    description: >-
+      Whether to use change detection for pull requests. If set to `true`, will
```

Comment:
Any reason not to have this on by default?

Reply:
It's a breaking change, and different what most collections do right now. Basically only the collections using AZP use change detection, all others simply run all tests.

Also switching this on effectively disables code coverage reporting for pull requests.

Reply:
> Also switching this on effectively disables code coverage reporting for pull requests.

Could you explore if codecov's carryforward feature could fix this?